permissions/plugin-xml-job-to-job-dsl.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "xml-job-to-job-dsl"
+paths:
+- "com/adq/jenkins/xml-job-to-job-dsl"
```

Comment:
OK https://github.com/jenkinsci/xml-job-to-job-dsl-plugin/blob/19ec417afa17cd78ef6da48f5d77836777c105c6/pom.xml#L15-L16